scripts/download_logs.sh

```diff
@@ -14,6 +14,8 @@ JUNIT_REPORT_DIR=${JUNIT_REPORT_DIR:-"reports/"}
 KUBE_CRS=(clusterdeployment infraenv agentclusterinstall agent)
 CAPI_PROVIDER_CRS=(agentmachine agentcluster cluster machine machinedeployment machineset)
 ENABLE_KUBE_API=${ENABLE_KUBE_API:-"false"}
+DEBUG_FLAGS=${DEBUG_FLAGS:-""}
```

Comment:
``DEBUG_FLAGS`` is too generic. What specifically does it debug?

Reply:
The IDE (pycharm/vscode)

Reply:
If we want to change it will do on another PR

Reply:
okay, but let's settle on the "next PR" (and not "another PR") :joy: